doc/us-en/setup/Quick-Install.md

```diff
@@ -0,0 +1,69 @@
+# Quick Install OpenEdge
+
+Compared to the previous version which using manual download installation, the latest version (version 0.1.5) OpenEdge supports package manager installation. With package manager, users can quickly install OpenEdge by simply typing a few commands at the terminal.
```

Comment:
Compared to the previous version which using manually download installation, the latest version (0.1.5) of OpenEdge supports command line installation. With this method, users can quickly install OpenEdge by simply typing a few commands with the terminal.

Reply:
Compared to manual download software in previous version, it supports installing OpenEdge through package manager in latest version(0.1.5). With this method, users can quickly install OpenEdge by simply typing a few commands at terminal.